### PR TITLE
icedid

### PR DESCRIPTION
**Pull request by tccontre**
## IcedID New Detections

- rundll32_create_remote_thread_to_a_process.yml
- rundll32_createremotethread_in_browser.yml
- suspicious_icedid_regsvr32_cmdline.yml
- suspicious_icedid_rundll32_cmdline.yml
- suspicious_rundll32_plugininit.yml
- office_application_spawn_regsvr32_process.yml
- rundll32_dnsquery.yml
- rundll32_process_creating_exe_dll_files.yml
- chcp_command_execution.yml
- create_remote_thread_in_shell_application.yml
- drop_icedid_license_dat.yml
- icedid_exfiltrated_archived_file_creation.yml
- sqlite_module_in_temp_folder.yml

## IceID stories
stories/Icedid.yml

## Modified detection yml file to support IcedID TTP's

- recon_avproduct_through_pwh_or_wmi.yml

## add icedId stories to existing detections

- previously_seen_command_line_arguments.yml
- eventvwr_uac_bypass.yml
- fodhelper_uac_bypass.yml
- mshta_spawning_rundll32_or_regsvr32_process.yml
- office_application_spawn_rundll32_process.yml
- office_document_executing_macro_code.yml
- office_product_spawning_mshta.yml
- registry_keys_used_for_persistence.yml
- schedule_task_with_rundll32_command_trigger.yml
- suspicious_regsvr32_register_suspicious_path.yml
- account_discovery_with_net_app.yml 
- nltest_domain_trust_discovery.yml
- winevent_scheduled_task_created_within_public_path.yml

### CODE REVIEW

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Rundll32 Create Remote Thread To A Process :white_check_mark:
- diskUsage: 172032
- runDuration: 1.837s
- scanCount: 32

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Rundll32 CreateRemoteThread In Browser :white_check_mark:
- diskUsage: 172032
- runDuration: 1.141s
- scanCount: 32

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Rundll32 DNSQuery :white_check_mark:
- diskUsage: 331776
- runDuration: 2.82s
- scanCount: 3156

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Rundll32 Process Creating Exe Dll Files :white_check_mark:
- diskUsage: 335872
- runDuration: 9.601s
- scanCount: 17049

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Suspicious IcedID Rundll32 Cmdline :white_check_mark:
- diskUsage: 577536
- runDuration: 1.988s
- scanCount: 39

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Suspicious Rundll32 PluginInit :white_check_mark:
- diskUsage: 483328
- runDuration: 49.532000000000004s
- scanCount: 97809

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Create Remote Thread In Shell Application :white_check_mark:
- diskUsage: 172032
- runDuration: 1.105s
- scanCount: 66

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Office Application Spawn Regsvr32 process :white_check_mark:
- diskUsage: 507904
- runDuration: 27.825s
- scanCount: 50887

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

CHCP Command Execution :white_check_mark:
- diskUsage: 585728
- runDuration: 21.181s
- scanCount: 37796

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

IcedID Exfiltrated Archived File Creation :white_check_mark:
- diskUsage: 172032
- runDuration: 1.111s
- scanCount: 10

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Drop IcedID License dat :white_check_mark:
- diskUsage: 176128
- runDuration: 1.28s
- scanCount: 5

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Sqlite Module In Temp Folder :x:
- diskUsage: 135168
- runDuration: 0.004s
- scanCount: 0

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Suspicious IcedID Regsvr32 Cmdline :white_check_mark:
- diskUsage: 589824
- runDuration: 50.538000000000004s
- scanCount: 98124

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Sqlite Module In Temp Folder :x:
- diskUsage: 135168
- runDuration: 0.003s
- scanCount: 0

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

Sqlite Module In Temp Folder :white_check_mark:
- diskUsage: 176128
- runDuration: 1.069s
- scanCount: 171

File: detections/endpoint/create_remote_thread_in_shell_application.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - SourceProcessId
+  - StartAddress
+  - EventCode
+  - Computer
+  security_domain: endpoint
+  impact: 70
+  confidence: 100
+  risk_score: 70
+  context:
+  - source: endpoint
```
Comment (patel-bhavin) [inline detections/endpoint/create_remote_thread_in_shell_application.yml]: can you change this to `Source:Endpoint` , caps on first letter and no space after :

Comment (tccontre) [reply detections/endpoint/create_remote_thread_in_shell_application.yml]: thanks @patel-bhavin. i already modify all context for those new detection with that format. including the Stage:

**Review by P4T12ICK — APPROVED** [review]

**Pull request merged**